tensorflow/lite/toco/graph_transformations/propagate_fixed_sizes.cc

```diff
@@ -1856,8 +1856,8 @@ void ProcessArgMinMaxOperator(Model* model, Op* op) {
   std::vector<int> output_dims;
 
   output_dims.reserve(input_dims.size() - 1);
-  for (int i = 0; i < input_dims.size(); ++i) {
-    if (i != axis) {
+  for (size_t i = 0; i < input_dims.size(); ++i) {
+    if ( static_cast<int>(i) != axis) {
```

Comment:
And here too. It looks weird to change type of loop counter and then do the static cast immediately below.